README.md

```diff
@@ -2,18 +2,18 @@
 
 ## Description
 
-This is a subordinate charm of [kubernetes-master][1], deployed as part of the
-[Charmed Kubernetes][2] and enables Kubernetes cluster to use Ceph as a
-storage. Currently supported storage options are `ceph-xfs` and `ceph-ext4`
+This is a subordinate charm of [kubernetes-control-plane][1], deployed as part of
+[Charmed Kubernetes][2] and enables a Kubernetes cluster to use Ceph as a
+storage backend. Currently supported storage options are `ceph-xfs` and `ceph-ext4`
 with `cephfs` being work in progress.
 
 **__Note:__** This charm creates various Kubernetes resources, including pods.
 Therefore it requires `kubernetes-master` to run in privileged mode (config
-option `allow-privileged=true`) 
+option `allow-privileged=true`)
 
 ## Usage
 
-Since this charm is not published in charmstore yet, it needs to be built from
+Since this charm is not published in charmhub yet, it needs to be built from
 source.
```

Comment:
This can be removed now that https://charmhub.io/ceph-csi should be working

```suggestion
Since this charm is not published in charmhub yet, it needs to be built from
source.
```